server/src/main/java/com/board/server/modules/member/AccountController.java

```diff
@@ -23,25 +26,46 @@
 @RestController
 public class AccountController {
 
+    private static final int FIRST_ERROR = 0;
+
     private final AccountService accountService;
     private final AccountMapper accountMapper;
+    private final SignUpValidator signUpValidator;
 
     @PostMapping("/members/sign-up")
-    public ResponseEntity<Void> signUp(@RequestBody @Valid SignUpRequestDto signupRequestDto)
+    public ResponseEntity<Void> signUp(@RequestBody @Valid SignUpRequestDto signupRequestDto, Errors errors)
             throws MessagingException {
+        signUpValidator.validate(signupRequestDto, errors);
+        if (errors.hasErrors()) {
+            throw new InvalidSignUpRequestException(getFirstErrorDefaultMessage(errors));
+        }
+
         accountService.signUp(signupRequestDto);
 
         return ResponseEntity.ok().build();
     }
 
+    private String getFirstErrorDefaultMessage(Errors errors) {
+        return errors.getAllErrors().get(FIRST_ERROR).getDefaultMessage();
```

Comment:
`FIRST_ERROR`가 이 안에서만 사용된다면 클래스 상수로 만들기보다는 이 메소드 스코프 안에서 처리하는 걸로 해도 괜찮습니다.
클래스 상수가 있다면 일단 개발자는 이것을 사용하는 일반적인 용도를 체크하려고 할 것입니다.
그런 비용을 줄일 수 있겠죠.

```suggestion
        final int firstError = 0;
        return errors.getAllErrors().get(firstError).getDefaultMessage();
```